cleiton-ferreira/1.ii/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Curriculum Vitae</title>
+    <link rel="stylesheet" href="https://www.w3schools.com/lib/w3.css">
+</head>
+<body>
+    <table align="center">
```

Comment:
Please don't use this attribute anymore `align="center"` it breaks the meaning of HTML. You can do the same with CSS.